transformer_nuggets/llama/train.py

```diff
@@ -15,7 +15,8 @@
 import numpy as np
 import torch
 from fire import Fire
-from float8_experimental.dynamic_linear import Float8DynamicLinear
```

Comment:
seems there is a name change in trunk: https://github.com/pytorch-labs/float8_experimental/blob/main/float8_experimental/float8_dynamic_linear.py

Reply:
totally was, thanks for fixing !